### ISSUE TITLE
Does not work in fullscreen application

### ISSUE THREAD

**Issue by emnoor** [wavexx/screenkey#17]
The overlay does not appear in full-screen applications like firefox or gnome-terminal.

```
➜ ~ lsb_release -a
LSB Version:    1.4
Distributor ID: Arch
Description:    Arch Linux
Release:    rolling
Codename:   n/a
➜ ~ gnome-shell --version
GNOME Shell 3.18.1
➜ ~ screenkey --version
0.7.0
```

**Comment by wavexx** [comment]
Indeed, it doesn't.

Could you try the latest sources?
I tested with a few window managers, but not gnome.

**Comment by emnoor** [comment]
It does work with latest sources in gnome.

**Comment by wavexx** [comment]
Perfect. If you could give it a few tests, I'm planning to release the current sources as 0.8 soon.

**Comment by wavexx** [comment]
Fixed in 0.8.
